rosdep/base.yaml

```
@@ -1199,6 +1199,12 @@ gperftools:
   fedora: [gperftools, gperftools-devel]
   gentoo: [dev-util/gperf]
   ubuntu: [google-perftools, libgoogle-perftools-dev]
+gphoto2:
+  arch: [gphoto2]
+  debian: [gphoto2]
+  fedora: [gphoto2]
+  gentoo: [gphoto2]
```

I think we need to include the category for Gentoo:
```suggestion
  gentoo: [media-gfx/gphoto2]
```